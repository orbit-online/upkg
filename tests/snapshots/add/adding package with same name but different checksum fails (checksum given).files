[drwxr-xr-x]  .
├── [drwxr-xr-x]  .upkg
│   ├── [drwxr-xr-x]  .bin
│   │   └── [lrwxrwxrwx]  acme-empty-v1.0.2.bin -> ../.packages/acme-empty@ef8a1bc060bb699dd758cfb1482d20af721f6a37ec5b79e489a4a51f8329626c/bin/acme-empty-v1.0.2.bin
│   ├── [drwxr-xr-x]  .packages
│   │   └── [drwxr-xr-x]  acme-empty@ef8a1bc060bb699dd758cfb1482d20af721f6a37ec5b79e489a4a51f8329626c
│   │       ├── [drwxr-xr-x]  bin
│   │       │   └── [-rwxr-xr-x]  acme-empty-v1.0.2.bin
│   │       └── [-rw-r--r--]  upkg.json
│   └── [lrwxrwxrwx]  acme-empty -> .packages/acme-empty@ef8a1bc060bb699dd758cfb1482d20af721f6a37ec5b79e489a4a51f8329626c
└── [-rw-r--r--]  upkg.json
