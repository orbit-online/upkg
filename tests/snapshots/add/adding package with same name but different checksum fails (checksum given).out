upkg: Already downloaded '$BATS_RUN_TMPDIR/package-fixtures/duplicates/acme-empty-v1.0.2-metadata.tar'
upkg: conflict: There is more than one package with the name 'acme-empty' ('acme-empty' and 'acme-empty')
upkg: An error occurred while installing '$BATS_RUN_TMPDIR/package-fixtures/default/acme-empty-v1.0.2-metadata.tar'
